### DISCUSSION TITLE
Serverless support: Passing outputs from one service to another

### DISCUSSION THREAD

**Discussion by tbehunin** [seed-run/serverless-seed#5] [Q&A]
Looking through all the docs, I can't seem to find how one goes about passing outputs from one service to another - similar to how serverless-compose works. Can this be configured somehow on seed.run?

**Comment by jayair** [comment]
We don't have a built in way for this in Seed right now.

**Reply by tbehunin** [reply]
Ok thanks for clarifying. Is this feature on the roadmap in the future and/or do you know of workarounds that people use to make this work?

**Reply by jayair** [reply]
Honestly we are in a bit of a wait and see mode with Serverless Framework. It doesn't look like it's being actively worked on. I think a work around would be to use CloudFormation exports?

**Comment by tbehunin** [comment]
I've employed a bit of a work-around by doing `${file(../other-service/serverless.yml)):someProp}` to pull in data from other serverless services. Seems to be working for my scenarios so far 🤞

**Comment by jayair** [comment]
Yup!
